README.md

```diff
@@ -10,7 +10,21 @@ It is a part of [Membrane Multimedia Framework](https://membraneframework.org).
 
 ## Installation
 
-Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
+First, you need to install [MAD](https://www.underbit.com/products/mad/) on your system.
+
+### macOS
+
+```bash
+brew install mad
+```
+
+### Ubuntu
+
+```bash
+sudo apt-get install libmad0-dev
+```
+
+Then, add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
```

Comment:
I would move the "Run `mix deps.get` after the code snippet with `{:membrane_audio_mix_plugin, "~> 0.15.2"}`